src/components/Services/DtsViewEdit.tsx

```diff
@@ -17,9 +17,9 @@ type ApiGitHub = {
   type: string;
 }
 type TemplateInfo = {
-  name: string; 
-  value: string; 
-  schema?:string|null;
+  name: string;
+  value: string;
+  schema?: string | null;
```

Comment:
Is it possible for the value to be null? Please review it, keep in mind that when the compiler shows you errors you have to correct them but it is not always the first option, in this case there must be more options since if you accept null values ​​the information in the template could not be correct